std::unordered_map compilation problem

Hello,

I am working with bde library and I was getting some compilation errors when I included std::unordered_map:

..../bdeExample1.cpp:24:9: error: ‘unordered_map’ in namespace ‘std’ does not name a type

 I have found this code in BDE_BUILD_PATH/include/bsl/unordered_map:

#if 0 // Native STL header disabled until we routinely test for C++11 *libs*
#if defined(BSLS_COMPILERFEATURES_SUPPORT_INCLUDE_NEXT)
#include_next <unordered_map>
#else
#include BSL_NATIVE_CPP_LIB_HEADER(unordered_map)
#endif
#endif

I commented the #if 0/#endif statement and it seems to work fine. Is there any bde library version with this issue fixed? or how could I perform the tests needed to make sure things are working properly?

--- a/groups/bsl/bsl+stdhdrs/unordered_map
+++ b/groups/bsl/bsl+stdhdrs/unordered_map
@@ -29,13 +29,11 @@ BSLS_IDENT("$Id: $")
 #   include <bsl_stdhdrs_incpaths.h>
 #   endif
 
-#if 0 // Native STL header disabled until we routinely test for C++11 *libs*
 #   if defined(BSLS_COMPILERFEATURES_SUPPORT_INCLUDE_NEXT)
 #     include_next <unordered_map>
 #   else
 #     include BSL_NATIVE_CPP_LIB_HEADER(unordered_map)
 #   endif
-#endif
 
 #else // defined(BSL_OVERRIDES_STD)
 

--- a/groups/bsl/bsl+stdhdrs/unordered_map.SUNWCCh
+++ b/groups/bsl/bsl+stdhdrs/unordered_map.SUNWCCh
@@ -25,9 +25,7 @@ BSLS_IDENT("$Id: $")
 #   include <bsl_stdhdrs_incpaths.h>
 #   endif
 
-#if 0 // Native STL header disabled until we routinely test for C++11 *libs*
 #   include BSL_NATIVE_CPP_LIB_HEADER(unordered_map)
-#endif
 
 #else
 

--- a/groups/bsl/bsl+stdhdrs/unordered_set
+++ b/groups/bsl/bsl+stdhdrs/unordered_set
@@ -29,13 +29,11 @@ BSLS_IDENT("$Id: $")
 #   include <bsl_stdhdrs_incpaths.h>
 #   endif
 
-#if 0 // Native STL header disabled until we routinely test for C++11 *libs*
 #   if defined(BSLS_COMPILERFEATURES_SUPPORT_INCLUDE_NEXT)
 #     include_next <unordered_set>
 #   else
 #     include BSL_NATIVE_CPP_LIB_HEADER(unordered_set)
 #   endif
-#endif
 
 #else // defined(BSL_OVERRIDES_STD)
 

--- a/groups/bsl/bsl+stdhdrs/unordered_set.SUNWCCh
+++ b/groups/bsl/bsl+stdhdrs/unordered_set.SUNWCCh
@@ -25,9 +25,7 @@ BSLS_IDENT("$Id: $")
 #   include <bsl_stdhdrs_incpaths.h>
 #   endif
 
-#if 0 // Native STL header disabled until we routinely test for C++11 *libs*
 #   include BSL_NATIVE_CPP_LIB_HEADER(unordered_set)
-#endif
 
 #else
 

